### DISCUSSION TITLE
Systemupdates across Releases and General Questions

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#347] [General]
> Once installed to the hard disk the system no longer has any connection with the SpiralLinux project, as it has been carefully designed to be completely dependent on the Debian project and not on a single developer. Therefore, users of existing SpiralLinux installations can upgrade to the latest Debian release as soon as it is available in the Debian repositories. Most aspects of the unique SpiralLinux configuration will be maintained across upgrades, but new SpiralLinux features will not be applied to existing installations as there are intentionally no package repositories created specifically for SpiralLinux. 

I have a question that, despite intensive searching and reading in all sources, I don't see answered.
Since SpiralLinux no longer has a connection to the project once it has been installed, how are the bug fixes from the new releases transferred to the installed system?

Example:
If I have 12.231001 installed, do I get the changes from 12.231008 and further 12.x releases?

And can I somehow tell in the system which release I'm using?

Thanks so much

**Comment by geckolinux** [comment]
Hi there,

> how are the bug fixes from the new releases transferred to the installed system?

They aren't. Usually bugs that are unique to SpiralLinux are a matter of missing packages or config file improvements that only affect a small subset of users, and those affected users can manually install additional packages or tweak a configuration as explained in the SpiralLinux release notes and/or bug report details.

The majority of bugs and subsequent fixes and improvements come directly from upstream Debian.

> And can I somehow tell in the system which release I'm using?

The only need for SpiralLinux version numbers is to differentiate between the SpiralLinux ISO version that was used to initially install it. Once installed you are using a customized Debian system with unmodified Debian packages and the `/etc/debian_version` and `/etc/os-release` files that Debian provides.

**Reply by AlphaElwedritsch** [reply]
Hi there Gecko
thank you very much for explanation.
Now, everything is clear for me.

And thank you very much for this fantastic (if not the best) Debian Spin.
The world needs more people like you
Keep it up and stay with us for a long time.

May the force be with you

Regards
Alpha

**Comment by AlphaElwedritsch** [comment]
Hi @geckolinux
May i ask you another question?

> VirtualBox support available out-of-the-box

What does this mean exactly?

Does it mean this:
>SpiralLinux includes a little-known official Debian add-on repository that provides VirtualBox officially from the Debian project. 

https://github.com/orgs/SpiralLinux/discussions/47#discussioncomment-3121102

**Reply by geckolinux** [reply]
Hi there, that's correct. Support running SpiralLinux as a guest under VirtualBox is installed by default, and users that need to use SpiralLinux as VirtualBox host to run VMs can also install it directly via Synaptic.

**Comment by AlphaElwedritsch** [comment]
Mal ganz einfach ausgesprochen:
Alter ist die Distro geil!
Ärgere mich, dass ich sie nicht schon früher entdeckt habe.

Da hab ich 1999 mit Debian angefangen, aber aus besagten Gründen (Benutzerfreundlichkeit Enduser) zu diversen anderen Distros gewechselt. Aber immer Debian oder SuSE basierend.
Am Ende bei Mint LMDE.

Aber jetzt mit SpiralLinux endlich wieder ein pures Debian mit Einfachheit. Spitze! ...
Ich freu mich....

-------


To put it simply:
Dude, the distro is awesome!
I'm annoyed I didn't discover them sooner.

I started with Debian in 1999, but switched to various other distros for the reasons mentioned (end-user user-friendliness). But always Debian or SuSE based.
In the end at Mint LMDE.

But now with SpiralLinux, finally a pure Debian with simplicity. Great! ...
I'm happy....

**Reply by geckolinux** [reply]
Thanks a lot for saying so! Really glad to hear that it's working well for your needs and preferences.

**Comment by AlphaElwedritsch** [comment]
Just as an aside. Do you actually use any of your spins yourself privately?
If so, which one with which DE?

**Reply by geckolinux** [reply]
Definitely! I almost always prefer Cinnamon.

**Reply by AlphaElwedritsch** [reply]
I think you as the gecko use the gecko spin, right? 
or do you take the path of enlightenment and use the SpiralLinux spin? ;)

cinamon is already cool. I used it on LMDE for a long time.
But for now I'm becoming friends with Budgie. It can also do something and is a nice alternative to the pure gnome.

**Reply by geckolinux** [reply]
I'm currently using SpiralLinux on all of my systems. I need my systems to just work and not change unexpectedly during updates. That was what openSUSE Leap offered, but since Leap is being deprecated I have migrated to SpiralLinux. For users that really do need a rolling release I still think that Tumbleweed / Slowroll (in the form of GeckoLinux) are the best rolling distros available, but for almost all users I think that a non-changing fixed-release system like Debian Stable is much more sensible.

**Comment by AlphaElwedritsch** [comment]
Hi there

> - Optimal power management with TLP preinstalled 

I wanna give feeddback regarding the power management

Works great on my HP EliteBook 840 G5

Better than before under LMDE cinnamon and much better than when I briefly tested fedora 38 with Gnome.

The battery consumption in standby mode is great. Before that, the battery ran out much more quickly.
In normal operation it is the same.

I'm currently using Budgie. I had never used it before. But I think that since Budgie is based on Gnome, the consumption should be a little higher than with cinnamon.

However. Your TLP implementation was great. Big praise here.

**Reply by geckolinux** [reply]
Excellent, good to hear. I didn't really adjust any of the TLP default settings, it's just included and enabled by default whereas many other distros don't use it. If you manually tweak it you could probably get even more efficiency.

**Reply by AlphaElwedritsch** [reply]
Will give it a try on weekend
Report it back here

**Reply by AlphaElwedritsch** [reply]
did these changes:

```
mkl@mkl-SpiralLinux:/etc$ sudo tlp-stat --cdiff
--- TLP 1.5.0 --------------------------------------------

+++ Configured Settings (only differences to defaults):
/etc/tlp.conf L0117: CPU_ENERGY_PERF_POLICY_ON_BAT="power"
/etc/tlp.conf L0136: CPU_BOOST_ON_AC="1"
/etc/tlp.conf L0137: CPU_BOOST_ON_BAT="0"
/etc/tlp.conf L0145: CPU_HWP_DYN_BOOST_ON_AC="1"
/etc/tlp.conf L0146: CPU_HWP_DYN_BOOST_ON_BAT="0"
/etc/tlp.conf L0169: PLATFORM_PROFILE_ON_AC="balanced"
/etc/tlp.conf L0170: PLATFORM_PROFILE_ON_BAT="low-power"
```


followed from here:
https://linrunner.de/tlp/index.html

The next few days will show what happens...
